### PR TITLE
Language code fix for file name

### PR DESCRIPTION
**Pull request by vedg**
Requirements added here: https://github.com/microsoft/vscode-dev/issues/477

Testing:

npm run compile-web
tested with portal route -
http://localhost:3000/powerplatform/portal/webpages/563a8e53-6cf7-ec11-82e4-000d3a8b4b81?orgUrl=https://org4e297bda.crm10.dynamics.com&dataSource=Dataverse&schema=portalDataverseoldschema&websiteId=24828659-6cf7-ec11-82e4-000d3a8b4b81&websiteName=Site%205&clientSessionId=20d30dc0-040a-11ed-a9c5-d5d08bd7e3e0&Client=PowerPagesStudio

image

### CODE REVIEW

File: src/web/client/common/remoteFetchProvider.ts [hunk 1]
```diff
@@ -49,10 +49,11 @@ export async function fetchData(accessToken: string, entity: string, entityId: s
     }
 }
 
-function createContentFiles(result: string, entity: string, queryParamsMap: any, entitiesSchemaMap: any, languageIdCodeMap: any, portalsFS: PortalsFS, dataverseOrgUrl: string, accessToken: string, entityId: string) {
+function createContentFiles(result: string, entity: string, queryParamsMap: any, entitiesSchemaMap: any, languageIdCodeMap: any, portalsFS: PortalsFS, dataverseOrgUrl: string, accessToken: string, entityId: string, websiteIdToLanguage: any) {
```
Comment (ashishchoudhary001) [inline src/web/client/common/remoteFetchProvider.ts:52]: I hope the types are addressed in the next PR?

**Review by ashishchoudhary001 — CHANGES_REQUESTED** [review]
Please change the default language code.

File: src/web/client/common/remoteFetchProvider.ts (hunk 1 above)
Comment (vedg) [reply src/web/client/common/remoteFetchProvider.ts:52]: yes its addressed in the lint fixes pr

File: src/web/client/common/remoteFetchProvider.ts [hunk 2]
```diff
@@ -49,10 +49,11 @@ export async function fetchData(accessToken: string, entity: string, entityId: s
...
     let languageCode = DEFAULT_LANGUAGE_CODE;
     if (languageIdCodeMap?.size) {
-        languageCode = languageIdCodeMap.get(queryParamsMap.get(WEBSITE_ID)) ? languageIdCodeMap.get(queryParamsMap.get(WEBSITE_ID)) : DEFAULT_LANGUAGE_CODE;
+        const lcid = websiteIdToLanguage.get(queryParamsMap.get(WEBSITE_ID))
+        languageCode = languageIdCodeMap.get(lcid) ? languageIdCodeMap.get(lcid) : DEFAULT_LANGUAGE_CODE;
```
[the remark this answers is not in the document]
Comment (vedg) [reply src/web/client/common/remoteFetchProvider.ts:56]: defaulted to blank for now

**Review by ashishchoudhary001 — APPROVED** [review]

File: src/web/client/common/authenticationProvider.ts [hunk 3]
```diff
@@ -22,7 +22,8 @@ export function getHeader(accessToken: string) {
 export async function dataverseAuthentication(dataverseOrgURL: string): Promise<string> {
     let accessToken = '';
     try {
-        const session = await vscode.authentication.getSession(PROVIDER_ID, [dataverseOrgURL + SCOPE_OPTION], { createIfNone: true });
+        //const session = await vscode.authentication.getSession(PROVIDER_ID, [dataverseOrgURL + SCOPE_OPTION], { createIfNone: true });
```
Comment (tyaginidhi) [inline src/web/client/common/authenticationProvider.ts:25]: remove the commented part?

Comment (vedg) [reply src/web/client/common/authenticationProvider.ts:25]: yes its removed in latest pr

**Pull request merged**